isbn/suche.html

```diff
@@ -281,37 +300,47 @@
 
 </head>
 <body>    
-    
+
 <h1>Unterstützungstool für Fachreferatsarbeit</h1>
 
 <a href="https://github.com/UB-Mannheim/malibu" target="_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="../img/fork-github.png" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png"></a>
 
-<table><tr><td>
-<form id="formIsbn" name='formular'  action="#">
-ISBN: <input id="eingabeIsbn" type="text" size="30" maxlength="30">
-<input id="submitIsbn" type="submit" value="Submit">
-</form></td><td><form id="formPpn" name='formular'  action="#">&nbsp; &nbsp; &nbsp; 
-ID: <input id="eingabePpn" type="text" size="30" maxlength="30">
-<input type="radio" name="eingabeVerbund" value="SWB" checked>SWB
-<input type="radio" name="eingabeVerbund" value="GBV">GBV
-<input type="radio" name="eingabeVerbund" value="B3KAT">B3KAT
-<input type="radio" name="eingabeVerbund" value="HEBIS">HEBIS
-<input type="radio" name="eingabeVerbund" value="HBZ">HBZ
-<input type="radio" name="eingabeVerbund" value="SWISS">SWISS
-<input type="radio" name="eingabeVerbund" value="OBVSG">OBVSG
-<input type="submit" value="Submit">
-</form>
-</td>  
-<td>&nbsp; &nbsp; &nbsp; <a href="http://wiki.bib.uni-mannheim.de/InterneSeiten/doku.php?id=anleitungen:anleitungen_und_formulare#fachreferenten" target="_blank" title="Internes Wiki -> Unterstützungstool für Fachreferatsarbeit">Hilfe</a></td>
-</tr></table> 
- 
-
-
+<table id="Suchzeile">
+	<tr>
```

Comment:
This looks like indentation using tab characters while the other lines use blanks (preferred).

Reply:
Hm... yes... do you want to fix it?